src/Wizard/Platform/Readline/ReadlinePlatform.ts

```diff
@@ -0,0 +1,147 @@
+import { Platform } from '../Platform';
+import { inject, injectable } from 'tsyringe';
+import { Helpers } from 'src/Utils';
+import { Tokens } from 'src/Wizard/Tokens';
+import { TestType } from 'src/Wizard/Models';
+import { ConnectivityService } from 'src/Wizard/Services/ConnectivityService';
+import readline from 'readline';
+
+@injectable()
+export class ReadlinePlatform implements Platform {
+  private rl: readline.Interface;
+  private delimiter: string = '\n\r--\n';
```

Comment:
Use [EOL](https://nodejs.org/api/os.html#os_os_eol)
```suggestion
  private readonly delimiter = '\n\r--\n';
```